interface-definitions/service_dns_forwarding.xml.in

```diff
@@ -793,6 +793,155 @@
+                          </valueHelp>
+                          <constraint>
+                            <validator name="url" argument="--scheme http --scheme https"/>
+                          </constraint>
+                        </properties>
+                      </leafNode>
+                    </children>
+                  </node>
+                  <node name="options">
+                    <properties>
+                      <help>Zone to cache options</help>
+                    </properties>
+                    <children>
+                      <leafNode name="timeout">
+                        <properties>
+                          <help>The maximum time a retrieval using the AXFR or URL source may take</help>
+                          <valueHelp>
+                            <format>u32:1-3600</format>
+                            <description>Request timeout in seconds</description>
+                          </valueHelp>
+                          <constraint>
+                            <validator name="numeric" argument="--range 1-3600"/>
+                          </constraint>
+                        </properties>
+                        <defaultValue>20</defaultValue>
+                      </leafNode>
+                      <leafNode name="refresh">
+                        <properties>
+                          <help>The interval to wait between retrievals. A value of zero means the retrieval is done once at startup and on Lua configuration reload</help>
+                          <valueHelp>
+                            <format>u32:0-2147483647</format>
+                            <description>Refresh period in seconds</description>
+                          </valueHelp>
+                          <constraint>
+                            <validator name="numeric" argument="--range 0-2147483647"/>
+                          </constraint>
+                        </properties>
+                        <defaultValue>86400</defaultValue>
+                      </leafNode>
+                      <leafNode name="retry-on-error">
+                        <properties>
+                          <help>The interval to wait before retrying a failed transfer</help>
+                          <valueHelp>
+                            <format>u32:1-2147483647</format>
+                            <description>Retry period in seconds</description>
+                          </valueHelp>
+                          <constraint>
+                            <validator name="numeric" argument="--range 1-2147483647"/>
+                          </constraint>
+                        </properties>
+                        <defaultValue>60</defaultValue>
+                      </leafNode>
+                      <leafNode name="max-received">
+                        <properties>
+                          <help>The maximum size of an update via the AXFR or URL source. The default value of 0 means no restriction</help>
```

Comment:
I'd just say "Maximum zone size in megabytes". Defaults can be found in the docs.

Reply:
done, about default from doc:

> The default value of 0 means no restriction.